### ISSUE TITLE
`netlify build` fails after upgrading to 2.47, works on 2.46 (Windows only, CLI only)

### ISSUE THREAD

**Issue by dalmo3** [netlify/cli#852]
**- Do you want to request a _feature_ or report a _bug_?**
Bug

**- What is the current behavior?**
```
┌─────────────────────────────┐
│        Netlify Build        │
└─────────────────────────────┘

> Version
  @netlify/build 0.3.10

> Flags
  mode: cli

> Current directory       
  D:\dev\projects\covidmap

> Config file
  No config file was defined: using default values.

> Context
  production

┌────────────────────────────────────┐
│ 1. build.lifecycle.onBuild command │
└────────────────────────────────────┘

$ npm run build
: not foundram Files/nodejs/npm: 3: /mnt/c/Program Files/nodejs/npm:
: not foundram Files/nodejs/npm: 5: /mnt/c/Program Files/nodejs/npm:
/mnt/c/Program Files/nodejs/npm: 6: /mnt/c/Program Files/nodejs/npm: Syntax error: word unexpected (expecting "in")

┌─────────────────────────────┐
│        Build failed         │
└─────────────────────────────┘

  Error message
  Command failed with exit code 2: npm run build

  Error location
  In configuration "build.lifecycle.onBuild" command:
  npm run build
```

**- If the current behavior is a bug, please provide the steps to reproduce.**
Just `netlify build`. 

The bug happens on multiple projects so I can't see it being due to any config files.

They all fail with that weird syntax error.

**- What is the expected behavior?**
I downgraded to `@2.46.0` and it's working fine again.

**- Local Environment Information**

```
────────────────────┐
 Environment Info   │
────────────────────┘

  System:
    OS: Windows 10 10.0.18363
    CPU: (8) x64 Intel(R) Core(TM) i7-6700HQ CPU @ 2.60GHz
  Binaries:
    Node: 13.3.0 - C:\Program Files\nodejs\node.EXE
    npm: 6.13.4 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Edge: 44.18362.449.0

netlify-cli/2.47.0 win32-x64 node-v13.3.0
```

**Comment by ehmicky** [comment]
Hi @dalmo3!

Thanks for reaching out.
Could you please share the link to your repository? I will try to reproduce this issue. Thanks!

**Comment by ehmicky** [comment]
The following PR #853 should fix your problem @dalmo3. We will release it soon.

**Comment by ehmicky** [comment]
We have released `2.48.0`, could you please try now?

**Comment by dalmo3** [comment]
All good now. Thanks a lot!
